### PR TITLE
SDAP-17: Working version of Ningester

### PR DESCRIPTION
**Pull request by fgreg**
This addresses SDAP-17

### CODE REVIEW

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```
Comment (lewismc) [inline docker/entrypoint.sh]: License header please.

File: docker/example_config/ConnectionSettings-DockerForMac.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (lewismc) [inline docker/example_config/ConnectionSettings-DockerForMac.yml]: License header please.

File: docker/example_config/ExampleJob.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Tile Slicer Config
```
Comment (lewismc) [inline docker/example_config/ExampleJob.yml]: License header please.

File: docker/install_nexusproto.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```
Comment (lewismc) [inline docker/install_nexusproto.sh]: License header please.

File: docker/install_ningester.sh [hunk 5]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```
Comment (lewismc) [inline docker/install_ningester.sh]: License header please.

File: docker/install_ningesterpy.sh [hunk 6]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```
Comment (lewismc) [inline docker/install_ningesterpy.sh]: License header please.

File: gradle/wrapper/gradle-wrapper.properties [hunk 7]
```diff
@@ -1,6 +1,5 @@
-#Thu Oct 12 17:50:39 PDT 2017
 distributionBase=GRADLE_USER_HOME
```
Comment (lewismc) [inline gradle/wrapper/gradle-wrapper.properties]: Can the **gradle/wrapper/gradle-wrapper.jar** file be downloaded as part of a build? Does it need to reside in SCM?

Comment (fgreg) [reply gradle/wrapper/gradle-wrapper.properties]: Needs to be in SCM. This is how the gradle wrapper works; the ./gradlew script calls the gradle-wrapper.jar which will download gradle if not already installed. It allows for anyone to checkout the project and run ./gradlew build without having to first install gradle.

Comment (lewismc) [reply gradle/wrapper/gradle-wrapper.properties]: ACK @fgreg

**Comment by fgreg** [comment]
@lewismc It does not allow me to merge at all anymore.

I followed the steps here: https://gitbox.apache.org/setup/ but I don't have a 'Merge' button and I no longer can push to the repo

`fatal: repository 'https://git-wip-us.apache.org/repos/asf/incubator-sdap-ningester.git/' not found`

**Comment by lewismc** [comment]
OK @fgreg I think this is just a permissions issue, I'll try to make good on the permissions issue today.

**Comment by ntquach** [comment]
Looks good to me.

**Pull request merged**